packages/lit-helpers/src/read-only-properties-mixin.js

```diff
@@ -0,0 +1,90 @@
+/** @typedef {import('lit-element').PropertyDeclaration & { readOnly: Boolean }} PropertyDeclarationWithReadOnly */
```

Comment:
```suggestion
/** @typedef {import('lit-element').PropertyDeclaration & { readOnly: Boolean }} ReadOnlyPropertyDeclaration */
```

Not a strong preference, but seems to make sense to me.